brambling/views/user.py

```diff
@@ -173,14 +156,7 @@ def get_context_data(self, **kwargs):
             },
             'stripe_live_settings_valid': stripe_live_settings_valid(),
             'stripe_test_settings_valid': stripe_test_settings_valid(),
-            'dwolla_live_settings_valid': dwolla_live_settings_valid(),
-            'dwolla_test_settings_valid': dwolla_test_settings_valid(),
         })
```

Comment:
Future thought: honestly weird that we give users access to test cards

Reply:
I think it's mostly for the demo user / event? But also - yes.